integr8ly/tutorial-web-app
# fix(about): replace Brand logo in About Modal

## Motivation
Replace the Brand logo in the About Modal from the individual red fedora, to the full Red Hat brand (red fedora + Red Hat). Fixes INTLY-2294

## What
Replace logo in About Modal

## Why
Incorrect Brand usage

## How
Update image

## Verification Steps

1. Go to the About Modal
2. See the new Brand image

## Checklist:


## Progress


## Additional Notes

**OLD**
<img width="1920" alt="Screen Shot 2019-06-17 at 10 15 59 AM" src="https://user-images.githubusercontent.com/4032718/59611932-3b1cc900-90ea-11e9-9e57-49c761fd4c11.png">

**NEW**
<img width="1920" alt="Screen Shot 2019-06-17 at 10 16 13 AM" src="https://user-images.githubusercontent.com/4032718/59611957-41ab4080-90ea-11e9-8b41-2f8c3215121d.png">
